client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/RestService.java

```diff
@@ -239,6 +243,37 @@ public Schema lookUpSubjectVersion(Map<String, String> requestProperties,
     return schema;
   }
 
+
+  public Schema lookUpSubjectVersion(String schemaString, String subject,
+                                     boolean lookupDeletedSchema)
+      throws IOException, RestClientException {
+    RegisterSchemaRequest request = new RegisterSchemaRequest();
+    request.setSchema(schemaString);
+    return lookUpSubjectVersion(request, subject, lookupDeletedSchema);
+  }
+
+  public Schema lookUpSubjectVersion(RegisterSchemaRequest registerSchemaRequest,
+                                     String subject,
+                                     boolean lookupDeletedSchema)
+      throws IOException, RestClientException {
+    return lookUpSubjectVersion(DEFAULT_REQUEST_PROPERTIES, registerSchemaRequest, subject,
+                                lookupDeletedSchema);
+  }
+
+  public Schema lookUpSubjectVersion(Map<String, String> requestProperties,
```

Comment:
I left the old method intact just so that we could test backward compatibility without the query param. Will look at options to possibly reduce any code.